src/seedu/addressbook/data/AddressBook.java

```diff
@@ -133,4 +133,11 @@ public int hashCode() {
         // use this method for custom fields hashing instead of implementing your own
         return Objects.hash(allPersons, allTags);
     }
+
+    /**
+     * Sorts all persons in the list according to their full name
+     */
+    public void sort(){
```

Comment:
coding style violation. [read this](https://oss-generic.github.io/process/codingStandards/CodingStandard-Java.html)